openebs/pkg/v1/snapshot_api.go

```diff
@@ -87,7 +88,7 @@ func (v OpenEBSVolume) CreateSnapshot(volName string, snapName string) (string,
 }
 
 // ListVolume to get the info of Vsm through a API call to m-apiserver
-func (v OpenEBSVolume) ListSnapshot(volName string, snapname string, obj interface{}) error {
+func (v OpenEBSVolume) ListSnapshot(volName string, snapname string, namespace string, obj interface{}) error {
```

Comment:
Is this function currently not used?

Reply:
Yes, snapshot list is just for the logging purpose , not really needed by snapshot-controller.